code/modules/research/experimentor.dm

```diff
@@ -170,28 +170,25 @@
 	if(!linked_console)
 		dat += "<b><a href='byond://?src=\ref[src];function=search'>Scan for R&D Console</A></b><br>"
 	if(loaded_item)
-		if(recentlyExperimented)
-			dat += "<b>The [src] is still resetting!</b>"

```

Comment:
Because it is annoying, and not really needed since the same cooldown applies when using the experimentation functions. It swaps the interface to the resetting mode, meaning you must use the reset button to get back to the experimentation interface. I can extend the timer to compensate for this, if you really require that.